extra-security/clipass/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=clipass
+PKGSEC=security
+PKGDEP="nodejs"
```

Comment:
I doubt that this package actually depends on `nodejs`. Could you please confirm?

Reply:
The execution of Clipass relies on `node` command as a runtime, and the building requires `npm`. These facts suggested me that `nodejs` should be listed as a dependency. Enlighten me if there is any misunderstanding.

Reply:
> The execution of Clipass relies on node command as a runtime, and the building requires npm. 

Got it, then you're right. Just drop `$BUILDDEP`.

Reply:
Line removed.